packages/lib-core/package.json

```diff
@@ -3,46 +3,33 @@
   "version": "1.0.0-alpha2",
   "description": "Allows loading, managing and interpreting plugins",
   "license": "Apache-2.0",
-  "type": "module",
   "repository": {
     "type": "git",
     "url": "https://github.com/openshift/dynamic-plugin-sdk.git",
     "directory": "packages/lib-core"
   },
-  "files": [
-    "dist/index.js",
-    "dist/index.d.ts"
-  ],
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
   "scripts": {
     "prepack": "yarn build",
     "prepublishOnly": "yarn test",
-    "build": "rm -rf dist && rollup -c",
+    "build": "rm -rf dist && yarn run -T rollup -c",
     "lint": "yarn lint:run --ext .ts,.tsx src rollup.config.js",
     "test": "echo 'Write some tests!'"
   },
   "peerDependencies": {
     "react": "^17.0.2",
+    "react-router": "^5.2.1",
     "redux": "^4.1.2"
   },
   "dependencies": {
     "lodash-es": "^4.17.21",
     "yup": "^0.32.11"
   },
-  "devDependencies": {
-    "@monorepo/common": "0.0.0-fixed",
-    "@rollup/plugin-node-resolve": "^13.1.1",
-    "@rollup/plugin-typescript": "^8.3.0",
-    "@types/lodash-es": "^4.17.5",
-    "@types/react": "^17.0.37",
-    "@types/react-router-dom": "^5.1.2",
-    "react": "^17.0.2",
-    "redux": "^4.1.2",
-    "rollup": "^2.61.1",
-    "rollup-plugin-analyzer": "^4.0.0",
-    "rollup-plugin-dts": "^4.0.1",
-    "tslib": "^2.3.1",
-    "typescript": "~4.4.4"
+  "peerDependenciesMeta": {
```

Comment:
I like the usage of this!

Reply:
@karelhala Thanks, the reason I've used it is because it allows devs to write simple plugins that use only a small subset of extension types, i.e. not necessarily extension types that depend on Redux etc.

This is assuming that plugins use TypeScript to declare their extensions, since `react-router` and `redux` dependencies here are utilized only via type imports.

Reply:
Yeah, exactly! I've been scratching my head why we mark them as optional and went looking at the code and realised we use just types out of these packages.